src/main/resources/logback.groovy

```diff
@@ -16,15 +16,14 @@
 
 import ch.qos.logback.classic.AsyncAppender
 import ch.qos.logback.classic.encoder.PatternLayoutEncoder
-import ch.qos.logback.core.Appender
-import ch.qos.logback.core.ConsoleAppender
-import ch.qos.logback.core.rolling.FixedWindowRollingPolicy
-import ch.qos.logback.core.rolling.RollingFileAppender
-import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy
+import ch.qos.logback.core.util.FileSize
+import com.nike.cerberus.util.ConfigUtils
 
 import static ch.qos.logback.classic.Level.*
 import static ch.qos.logback.core.spi.ContextAware.addInfo
 
+System.setProperty("logback.debug", "true")
```

Comment:
We don't want debug do we?

Reply:
I just enabled this so I can get more info in dev2 and bake an AMI

Reply:
Are you removing then?

Reply:
After perf testing, you have to have this enabled if you want any sort of logging or std out/err from these classes.